Project 1 Day 2/application.py

```diff
@@ -16,11 +16,38 @@
 # Configure session to use filesystem
 app.config["SESSION_PERMANENT"] = False
 app.config["SESSION_TYPE"] = "filesystem"
+app.config['SQLALCHEMY_DATABASE_URI'] = 'postgres://sgrwbvfpglusrx:579f0fd0edc0214e2642000c00ec1734a0206342eb055c5bf214596697441880@ec2-18-213-176-229.compute-1.amazonaws.com:5432/d5na60isuu66op'
+
 Session(app)
 
 # Set up database
 engine = create_engine(os.getenv("DATABASE_URL"))
-db = scoped_session(sessionmaker(bind=engine))
+Session = scoped_session(sessionmaker(bind=engine))
+session = Session()
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+db = SQLAlchemy(app)
+
+
+#TASK - 4 CREATING A DATABASE TABLE 
+class Dataentry(db.Model):
+    __tablename__ = "dataentry"
+    # IndexNo = db.Column(db.Integer)
+    Name = db.Column(db.String() , nullable=False)
+    Email = db.Column(db.String(), primary_key=True, nullable=False)
+    password = db.Column(db.String() , nullable=False)
+    timezone = db.Column(db.DateTime(timezone=True), nullable=False)
+    def __init__ (self, name, email, password, time):
+        # self.IndexNo = sno
+
+        self.Name = name
+        self.Email = email
+        self.password = bcrypt.encrypt(password)
+        self.timezone = time
```

Comment:
You may use the datetime.now function here to assign the value for time.

BTW,  timezone is  not a good variable name in this case.